packages/playground/src/weblets/tf_contracts_list.vue

```diff
@@ -260,11 +279,35 @@ async function onDelete() {
   }
   deleting.value = false;
 }
+
+async function getNodeStatus(nodeIDs: (number | undefined)[]) {
+  const resultPromises = nodeIDs.map(async nodeId => {
+    if (typeof nodeId !== "number") return {};
+    const status = (await gridProxyClient.nodes.byId(nodeId)).status;
+    return { [nodeId]: status };
+  });
+
+  const resultsArray = await Promise.all(resultPromises);
+
+  return resultsArray.reduce((acc, obj) => Object.assign(acc, obj), {});
+}
+function getNodeStateColor(state: NodeStatus): string {
```

Comment:
Set a default case or the return type might be undefined.

Reply:
the nodeStatus have only three cases and each case is handled, i believe there is no need to set default case as long as we didn't have any unhanded NodeStatus value, what do you think ?